SKILL.tsv

```diff
@@ -1775,7 +1775,7 @@ SKILL_20150317_001774	HP Aurora
 SKILL_20150317_001775	HP recovery is improved.
 SKILL_20150317_001776	Size has shrunk.
 SKILL_20150317_001777	Parrot
-SKILL_20150317_001778	Ignore evasion of enemy but chance of critical and critical evasion is decreased.
+SKILL_20150317_001778	Your attacks are guaranteed to hit, increasing your Critical Rate and Critical Resistance.
```

Comment:
This is the text that showed up under aggressor when it was being used.  Changing this line also changed that text.  I can change it back, but since aggressor has already been removed, I'm not sure if it matters now?